### ISSUE TITLE
Make ES6 the default output for the playground

### ISSUE THREAD

**Issue by jfrolich** [rescript-association/rescript-lang.org#56]
It's probably what 90% of the people would be using (isn't it the defacto standard for NodeJS as well at this point?).

Also it would be great to store the settings in localstorage, so they are restored on refresh

**Comment by ryyppy** [comment]
> It's probably what 90% of the people would be using (isn't it the defacto standard for NodeJS as well at this point?).

my impression was that ppl are still very heavily relying on commonjs (also for webpack setups etc)... i haven't been using any node that allowed ES6 imports so far (still relying on the `esm` module), but I might be out of touch with what everyone is doing nowadays

> Also it would be great to store the settings in localstorage, so they are restored on refresh

I wanted to hold off from local storage stuff after we collected some feedback for the new share-link feature. Local storage always means maintaining some kind of versioning schema to prevent problems with different bundle versions, also handle specific error scenarios, e.g. I can remember a time where the ReasonML playground couldn't boot up because I was loading malformed code from storage (ending in an endless loop).

Definitely something to look into at some point

**Comment by jfrolich** [comment]
> Edit: According to the NodeJS docs, ES6 are still in experimental phase
Ok indeed experimental but it is enabled by default. I don't use node heavily but I did saw some projects pick up modules (eg Gatsby).

> Local storage always means maintaining some kind of versioning schema
I would suggest each particular setting into a specific local storage key, so you don't need anything like that. I agree that keeping the code itself in local storage is potentially problematic (I experienced this with the reasonml playground multiple times as well).
